File: app/src/main/java/com/neome/java/api/ent/base/dto/DtoEntActionExecuteWorkflow.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdAutomation;
import com.neome.java.api.meta.base.Types.MetaIdForm;
import com.neome.java.api.meta.base.dto.EntVdWorkflowPointer;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DtoEntActionExecuteWorkflow extends DtoEntAction
{
  @Nullable
  public MetaIdForm automationFormId;

  @Nullable
  public MetaIdAutomation automationId;

  @Nullable
  public EntVdWorkflowPointer workflowPointer;
}

File: app/src/main/java/com/neome/api/ent/base/dto/DtoEntActionExecuteWorkflow.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.base.dto

import com.neome.api.ent.base.dto.DtoEntAction
import com.neome.api.meta.base.dto.EntVdWorkflowPointer
import com.neome.api.meta.base.Types.MetaIdAutomation
import com.neome.api.meta.base.Types.MetaIdForm

interface DtoEntActionExecuteWorkflow : DtoEntAction
{
  val automationFormId: MetaIdForm?
  val automationId: MetaIdAutomation?
  val workflowPointer: EntVdWorkflowPointer?
}